code/style.css

```diff
@@ -2,6 +2,651 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  color: #ffffff90;
+  background-color: #000;
+  margin: 0;
+  font-family: raleway;
+  font-weight: 200;
+}
+
+p {
+  margin-top: 0;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  text-align: center;
+}
+
+h1 {
+  font-size: 64px;
+  font-family: raleway dots;
+  font-weight: 100;
+  margin: 0;
+}
+
+/* CODE FOR ANIMATION ON BREATHE */
+
+.pulse {
+	transform: scale(1);
+	animation: pulse 4s infinite;
+}
+
+@keyframes pulse {
+	0% {
+		transform: scale(0.95);
+	}
+
+	70% {
+		transform: scale(1.05);
+	}
+
+	100% {
+		transform: scale(0.95);
+	}
+}
+
+/* -------- */
+
+
+h2 {
+  font-size: 1.125rem;
+  line-height: 1.5;
+  font-family: raleway;
+  font-weight: 200;
+  margin-left: 1rem;
+}
+
+h3 {
+  font-size: 1rem;
+  font-family: raleway;
+  font-weight: 200;
+}
+
+/* NAV & HERO */
+
+.navbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 0;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.company-logo {
+  width: 80px;
+  height: auto;
+}
+
+.hero-image {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url("unsplash_pic.jpg");
+  background-position: center;
+  background-size: cover;
+  position: relative;
+  height: 360px;
+}
+
+.hero-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 5rem;
+}
+
+
+.hero-button {
+  border-radius: 24px;
+  background-color: aqua;
+  font-family: raleway;
+  font-size: 1.125rem;
+  border: 0;
+  height: 48px;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  cursor: pointer;
+}
+
+.hero-button:hover {
+  opacity: 80%;
+}
+
+
+/*INFO */
+
+.sectionHeading {
+  margin-left: 0;
+  font-weight: 600;
+  text-align: center;
+}
+
+.middleSection {
+  display: flex;
+  flex-direction: column;
+}
+
+.preamble {
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-top: 0;
+}
+
+.preamble p {
+  font-size: 1.125rem;
+  line-height: 1.8rem;
+  margin-top: 0;
+}
+
+
+.jes-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  margin-top: 3rem;
```

Comment:
Can it be like "margin : 3rem 1rem 0 1rem;" ?

Reply:
True!! Will remember that in future coding! :-)